purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -48,6 +51,10 @@ open class PurchasesConfiguration(builder: Builder) {
             this.store = store
         }
 
+        fun dangerousSetting(dangerousSetting: DangerousSetting?) = apply {
+            this.dangerousSetting = dangerousSetting
+        }
```

Comment:
Should this be an array to future proof this if there are more `DangerousSetting`s that are needed in the future?

Reply:
Agreed. I actually started doing that and changed it to a single object, but I think the list is better.